### PR TITLE
Adds Holy Water Darts to Cargo

### PR DESCRIPTION
**Pull request by ItsMarmite**
## What Does This PR Do
Adds a new crate to Cargo, containing a new type of shotgun shell- holy water darts. These darts contain 10u of holy water per dart. The crate itself costing 400 credits for two boxes, the same as tranquilisers.

## Why It's Good For The Game
Vampires were balanced with the usage of holy water offensively by Security in mind, however since the removal of the RSG from RND- Security has had to turn to shotgun darts to make use of this intended counter to vampires. These darts, which contain 20u less holy water than shotgun darts should allow for Security to still make use of this advantage without significantly over-watering the vampires they come across.

## Testing
Ordered a crate, opened it, shot it, health scanned the skrell. Correct holy water amount.

image

## Changelog
:cl:
add: Adds 10u Holy Water dart boxes as cargo orderables.
/:cl:

### CODE REVIEW

**Comment by Erikos66** [comment]
Makes sense to have this, as said syringes/reagent shells aren't easily accessible. Having NT made shells for this exact purpose feels better than making them yourself.

I also love that gives more use to the shotgun 👍

**Comment by TravisAngeI** [comment]
prepare to receive some holy spirit, bitch
- holy inquisitor charles moments before blowing up the door to a cultist base

**Comment by Dafrek** [comment]
With how strong a single shot of holy water will be when compared to something like the immolator, it may need a bit of tweaking, similar to how https://github.com/ParadiseSS13/Paradise/discussions/16268 removed some of its more oppressing aspects because of the predicted increase of offensive holy water use against vampires.

Having a more easily accessible weapon that limits powerful vampires steamrolling sec by stealing their weapons and using it against them will be nice, especially if its slightly weaker overall than most other anti-vamp guns.

**Comment by spacegeese** [comment]
> prepare to receive some holy spirit, bitch -holy inquisitor charles moments before blowing up the door to a cultist base

image

File: code/modules/projectiles/ammunition/ammo_casings.dm [hunk 1]
```diff
@@ -306,6 +306,16 @@
...
+/obj/item/ammo_casing/shotgun/holy
+	name = "holy water darts"
+	desc = "A 12 gauge dart shell loaded with holy water."
+	icon_state = "hshell"
+	projectile_type = /obj/item/projectile/bullet/dart/syringe/holy
+	muzzle_flash_strength = MUZZLE_FLASH_STRENGTH_NORMAL
+	muzzle_flash_range = MUZZLE_FLASH_RANGE_NORMAL
+	materials = list(MAT_METAL=250)
+
+
```
Comment (Burzah) [inline code/modules/projectiles/ammunition/ammo_casings.dm:318]:
```suggestion
```

File: code/modules/projectiles/projectile/bullets.dm [hunk 2]
```diff
@@ -269,6 +269,12 @@
 	..()
 	reagents.add_reagent("haloperidol", 15)
 
+/obj/item/projectile/bullet/dart/syringe/holy
+
+/obj/item/projectile/bullet/dart/syringe/holy/New()
+	..()
+	reagents.add_reagent("holywater", 10)
```
Comment (Sirryan2002) [inline code/modules/projectiles/projectile/bullets.dm:276]: does this need to be in New() or can you put it into Initialize()???

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

**Review by Sirryan2002 — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Pull request merged**